blkn/src/FieldFeedback/index.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from "react";
+import Typography from "./../Typography";
+
+type Props = {
+  error?: string,
+  help?: string,
```

Comment:
I would use `React.Node` here. According to the design, the text may be formatted: https://zpl.io/VDDzogO

Reply:
That's a great way to solve it, thanks.

Accoding to zeplin mockups `error`s are not formatted in any case. But maybe we could also set it as `React.Node` to be on the safe side?

Reply:
Agree, let's make it both React.Node to make it consistent.